config/initializers/sidekiq.rb

```diff
@@ -0,0 +1,4 @@
+if ENV['SIDEKIQ_TESTING'] == 'inline'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.